core/designsystem/src/main/java/com/acon/core/designsystem/component/button/AconButton.kt

```diff
@@ -0,0 +1,60 @@
+package com.acon.core.designsystem.component.button
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.acon.core.designsystem.theme.AconTheme
+
+@Composable
+fun AconButton(
+    backGroundColor: Color,
+    borderColor: Color = Color.Transparent,
+    modifier: Modifier = Modifier,
+    borderWidth: Dp = 0.dp,
+    cornerRadius: Dp = 0.dp,
+    paddingHorizontal: Dp = 0.dp,
+    paddingVertical: Dp = 0.dp,
```

Comment:
```suggestion
    contentPadding: PaddingValues = PaddingValues(horizontal = 0.dp, vertical = 0.dp)
```
PaddingValues라는 타입으로 한 줄에 쓸 수 있습니다!!